### PR TITLE
Extend explanation of the functions

### PR DESCRIPTION
**Pull request by tschmetzer**
Available function comments were extended according to my understanding. 
Please check for correctness.

### CODE REVIEW

**Comment by tschmetzer** [comment]
Any clue what flake8 is complaining about?

File: python/plugins/processing/script/ScriptTemplate.py [hunk 1]
```diff
@@ -44,7 +44,10 @@ class ExampleProcessingAlgorithm(QgsProcessingAlgorithm):
 
     def tr(self, string):
         """
-        Returns a translatable string with the self.tr() function.
+        Returns a string translated by the self.tr() function.
+
+        This function makes use of the QT5 ability to translate a given
```
Comment (Gustry) [inline python/plugins/processing/script/ScriptTemplate.py:49]:
```suggestion
        This function makes use of the QT5 ability to translate a given
```

```
./python/plugins/processing/script/ScriptTemplate.py:49:72: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:60:75: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:61:81: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:62:80: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:84:78: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:85:80: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:86:79: W291 trailing whitespace
```

and so on for other lines @tschmetzer

Comment (tschmetzer) [reply python/plugins/processing/script/ScriptTemplate.py]: > 
> 
> ```
> ./python/plugins/processing/script/ScriptTemplate.py:49:72: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:60:75: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:61:81: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:62:80: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:84:78: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:85:80: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:86:79: W291 trailing whitespace
> ```
> 
> and so on for other lines @tschmetzer.
> 
> You must remove whitespaces at the end of each lines.

Thanks, what a shame! I was mislead by the error message. I'll never forget what a trailing whitespace is :D

File: python/plugins/processing/script/ScriptTemplate.py [hunk 2]
```diff
@@ -44,7 +44,10 @@ class ExampleProcessingAlgorithm(QgsProcessingAlgorithm):
...
```
Comment (nyalldawson) [inline python/plugins/processing/script/ScriptTemplate.py:49]:
```suggestion
        This function makes use of the Qt library's ability to translate a given
```

File: python/plugins/processing/script/ScriptTemplate.py [hunk 3]
```diff
@@ -44,7 +44,10 @@ class ExampleProcessingAlgorithm(QgsProcessingAlgorithm):
...
+        text strings into local languages.
```
Comment (nyalldawson) [inline python/plugins/processing/script/ScriptTemplate.py:50]:
```suggestion
        text string into local languages.
```

File: python/plugins/processing/script/ScriptTemplate.py [hunk 4]
```diff
@@ -53,11 +56,11 @@ def createInstance(self):
 
     def name(self):
         """
-        Returns the algorithm name, used for identifying the algorithm. This
-        string should be fixed for the algorithm, and must not be localised.
-        The name should be unique within each provider. Names should contain
-        lowercase alphanumeric characters only and no spaces or other
-        formatting characters.
+        Returns the algorithm name used for identifying the algorithm. This
+        string should be fixed for the algorithm and is not required to be
```
Comment (nyalldawson) [inline python/plugins/processing/script/ScriptTemplate.py:60]: In my opinion "localised" is a clearer phrasing then "in local language". But perhaps a compromise would be "and must not be translated"?

**Comment by tschmetzer** [comment]
Does anybody know why 7 checks haven't been executed? What's missing?

**Comment by roya0045** [comment]
I'd suggest rebase and wait.

**Comment by tschmetzer** [comment]
I rebased the branch again just now. The git tool suggests now to open a new PR. 
What's the correct workflow here? Can we reopen this PR or should I open a new PR?

**Comment by elpaso** [comment]
> I rebased the branch again just now. The git tool suggests now to open a new PR.
> What's the correct workflow here? Can we reopen this PR or should I open a new PR?

@tschmetzer no, I've reopened the current PR.

**Comment by tschmetzer** [comment]
> 
> 
> > I rebased the branch again just now. The git tool suggests now to open a new PR.
> > What's the correct workflow here? Can we reopen this PR or should I open a new PR?
> 
> @tschmetzer no, I've reopened the current PR.

Great, thanks!

**Pull request closed without merging**